ReactWindows/ReactNative.Net46/Views/ReactCanvas.xaml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+
+namespace ReactNative.Views
+{
+    public partial class ReactCanvas : UserControl
```

Comment:
>ReactCanvas [](start = 25, length = 11)

Can we push this out to a separate PR for a more holistic look at this approach (to support things like embedded views, etc.)